CHANGELOG.next.asciidoc

```diff
@@ -124,6 +124,7 @@ https://github.com/elastic/beats/compare/v8.7.1\...main[Check the HEAD diff]
 - Fix crash when loading azurewebstorage cursor with no partially processed data. {pull}35433[35433]
 - RFC5424 syslog timestamps with offset 'Z' will be treated as UTC rather than using the default timezone. {pull}35360[35360]
 - [system] sync system/auth dataset with system integration 1.29.0. {pull}35581[35581]
+- [Gcs Input] - Enhancement and bugfix for concurrency issues, flakey tests and context timing out. {pull}35605[35605]
```

Comment:
```suggestion
- [GCS Input] - Enhancement and bugfix for concurrency issues, flakey tests and context timing out. {pull}35605[35605]
```
but I think these should be broken out into separate lines that clearly describe each change (probably the enhancement should be a separate PR).

Reply:
This doesn't explain what is the enhancement and what is the bugfix. Ideally when there is a subtle bugfix, there should be no concomitant enhancememt.

Reply:
Sure, I'll break it into different statements. As for the PR I agree they should be separate usually but since this was more of a minor semantic change of passing the context instead of storing in structs and re enabling the tests in addition to the bug fixes, I thought I'd keep them in so it can be reviewed in one go.